napalm_onyx/onyx_ssh.py

```diff
@@ -247,16 +253,58 @@ def _get_flash_size(self):
         raise NotImplementedError("get_flash_size is not supported yet for onyx devices")
 
     def _enough_space(self, filename):
-        raise NotImplementedError("enough_space is not supported yet for onyx devices")
-
-    def _verify_remote_file_exists(self, dst, file_system='bootflash:'):
-        command = 'dir {0}/{1}'.format(file_system, dst)
-        output = self.device.send_command(command)
-        if 'No such file' in output:
-            raise ReplaceConfigException('Could not transfer file.')
+        """Return the free bytes on the host under /var/."""
+        command = 'show files system'
+        output = self._send_command(command)
+        bytes_free = '0'
+        matches = re.findall(r'Space Free\s+(\d+) MB', output)
+        if matches and len(matches) >= 2:
```

Comment:
not a good solution
what is a new fs added in the future or /config removed.
it would be better to use json and choose the one with the /var

Reply:
yes but even the json is not giving me any mapping, some thine like this 

```
 {
        "Space Free": "7989 MB",
        "Space Used": "1889 MB",
        "Space Percent Free": "80%",
        "header": "Statistics for /var filesystem",
        "Space Available": "7481 MB",
        "Inodes Percent Free": "99%",
        "Space Total": "9878 MB"
    }
```

may I check the header attribute if this object.header contains ('/var') take free size